### PR TITLE
Document ability to do object count quota for all namespaced resources

### LINKED ISSUE #7013: resource quota: document the ability to do object count quota for all namespaced resources
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

https://github.com/kubernetes/kubernetes/pull/54320 was merged in 1.9 and gives the ability to do object count quotas on all standard resources using the `count/<resource>.<group>` syntax and without writing code.

This is an important change and should be documented at https://kubernetes.io/docs/concepts/policy/resource-quotas/.

**Proposed Solution:**

I'll create a PR adding more details.

**Page to Update:**

- https://kubernetes.io/docs/concepts/policy/resource-quotas/
- https://kubernetes.io/docs/tasks/administer-cluster/quota-api-object/
- https://kubernetes.io/docs/tasks/administer-cluster/quota-pod-namespace/

/cc @sttts @derekwaynecarr

### PR DESCRIPTION
**Pull request by nikhita**
Fixes #7013

Relevant PR: https://github.com/kubernetes/kubernetes/pull/54320.

/cc derekwaynecarr deads2k sttts

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7441)

### CODE REVIEW

**Comment by nikhita** [comment]
This feature was introduced in 1.9 so this PR is against master, instead of the `release-1.10` branch.

File: docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -108,6 +108,9 @@ are supported:
 | `services.nodeports` | The total number of services of type node port that can exist in the namespace. |
 | `secrets` | The total number of secrets that can exist in the namespace. |
 
+Additionally, generic object count quota is supported on all standard resources
+using the `count/<resource>.<group>` syntax.
```
Comment (nikhita) [inline docs/concepts/policy/resource-quotas.md]: @derekwaynecarr could you elaborate on the `count/*` syntax?

File: docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -218,6 +221,30 @@ services                0       10
 services.loadbalancers  0       2
 ```
 
+Kubectl also supports object count quota for all namespaced resources
+using the syntax `count/<resource>.<group>`:
+
+```shell
+$ kubectl create namespace myspace
```
Comment (heckj) [inline docs/concepts/policy/resource-quotas.md]: please remove the `$` characters prefixing the commands (per style guide: https://kubernetes.io/docs/home/contribute/style-guide/#dont-include-the-command-prompt)

**Comment by derekwaynecarr** [comment]
I had written this but it looks like it never merged.

https://github.com/kubernetes/website/commit/6ef287a081adf03971f8cfb6d12a37d45e517b61

**Comment by nikhita** [comment]
> I had written this but it looks like it never merged.

Weird, let me incorporate those changes in this PR then. Thanks!

**Comment by nikhita** [comment]
@heckj @derekwaynecarr updated. ptal.

**Comment by heckj** [comment]
thanks @nikhita 
/approve

**Comment by heckj** [comment]
@derekwaynecarr or @sttts  - please review for technical accuracy. If you're happy with it please give it an `/lgtm` comment - that will merge it

**Pull request merged**